docs/topics/i18n/translation.txt

```diff
@@ -70,6 +70,16 @@ as a shorter alias, ``_``, to save typing.
        global ``_()`` function causes interference. Explicitly importing
        ``ugettext()`` as ``_()`` avoids this problem.
 
+    .. admonition:: What functions may be aliased as ``_``?
+
+        Only :func:`~django.utils.translation.gettext`,
+        :func:`~django.utils.translation.gettext_lazy`,
+        :func:`~django.utils.translation.ugettext` and
```

Comment:
I don't think we should add , here (English is not my first language but it seems wrong to me).